npm/package.json

```diff
@@ -39,17 +39,19 @@
   },
   "homepage": "https://github.com/nationalarchives/tdr-transfer-frontend#readme",
   "dependencies": {
+    "@nationalarchives/file-information": "1.0.4",
+    "@nationalarchives/tdr-generated-graphql": "1.0.12",
+    "@types/mocha": "^7.0.2",
```

Comment:
Is there a reason that @types/mocha and ajv dependencies have been added

Reply:
I added that as my jest types were not being recognised by vscode, but I can remove them as it works now.